### PR TITLE
Deletes object associated with given key from S3 compatible blob store

### PR DESCRIPTION
**Pull request by kalambet**
[#113462943](https://www.pivotaltracker.com/projects/956238/stories/113462943)

Signed-off-by: Yulia Gaponenko <yulia.gaponenko@ru.ibm.com>

### CODE REVIEW

File: src/s3cli/s3cli/main.go [hunk 1]
```diff
@@ -56,6 +66,12 @@ func main() {
 
 		defer dstFile.Close()
 		err = blobstoreClient.Get(src, dstFile)
+	case "del":
```
Comment (cppforlife) [inline src/s3cli/s3cli/main.go]: let's call this "delete".

**Comment by cppforlife** [comment]
Can you also add happy path delete scenario somewhere in https://github.com/pivotal-golang/s3cli/blob/master/ci/assets/generic-examples-template.bats

**Comment by cppforlife** [comment]
Also does delete behave in an idempotent way?

**Comment by kalambet** [comment]
Yes, the `delete` command behaviour is idempotent. Also I've added two tests, the first one for checking that `detele` is actual deletes and second that checks the idempotent behaviour of the command.

**Pull request merged**